### PR TITLE
Fixed version constraints

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "league/commonmark": "dev-master"
```
Comment (mnapoli) [inline composer.json:18]: Probably better to be forward compatible: `~0.7`?

File: composer.json [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
+        "league/commonmark": "0.7.*"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.3",
-        "codeclimate/php-test-reporter": "~0.1"
```
Comment (mnapoli) [inline composer.json:22]: It's probably also good to keep the current constraint to be forward compatible

File: composer.json (hunk 1 above)
Comment (colinodell) [inline composer.json:18]: The jump to `0.8` would be considered a major release which could break backwards compatibility, so `0.7.*` would be the safe constraint to use.  `~0.7` seems like a good compromise between that and `dev-master` though.

Comment (mnapoli) [inline composer.json:18]: Yeah that's true that we are in the `0.*` range, makes sense!

File: composer.json (hunk 2 above)
Comment (GrahamCampbell) [inline composer.json:22]: Again, I disagree. 0.2 could be a breaking release. The maintainers are free to re-namespace for example, which would trash everything...

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json:18]: I disagree. The tests are already failing due to breaking changes from 0.5->0.7...

We need to keep the minor version locked.

Comment (henriquemoody) [inline composer.json:18]: I agree with Campbell, probably `0.x.*` or `~0.x.0` are better.

Comment (henriquemoody) [inline composer.json:18]: @GrahamCampbell, will you change it to `0.4.*` instead just to make the current state stable, so?

Comment (mnapoli) [inline composer.json:18]: I think we all agree then ;)

File: composer.json (hunk 2 above)
Comment (mnapoli) [inline composer.json:22]: For such a tool I wouldn't fear that as they would break a lot of packages.

**Comment by henriquemoody** [comment]
The exact version is on commit  4cf7b51, however everything was fixed on #3 and we didn't need to change `codeclimate/php-test-reporter` constraints.

**Pull request closed without merging**